packages/@sanity/vision/src/components/VisionGui.js

```diff
@@ -273,7 +275,7 @@ class VisionGui extends React.PureComponent {
             <label className={styles.datasetSelectorContainer}>
               <span className={styles.datasetLabel}>Dataset</span>
               <Select
-                value={this.state.dataset || client.config().dataset}
+                value={this.state.dataset || client.config().dataset || this.props}
```

Comment:
Is passing `this.props` as value to Select intentional?

Reply:
:open_mouth: well of course not. this is what happens when you PR at 3AM... it's leftover code, must have glanced and thought there was only a formatting change there. can disregard this line.